tests/dvslib/dvs_vlan.py

```diff
@@ -46,6 +46,10 @@ def remove_vlan_member(self, vlanID, interface):
         member = "Vlan{}|{}".format(vlanID, interface)
         self.config_db.delete_entry("VLAN_MEMBER", member)
 
+    def remove_vlan_interface(self, vlanID):
```

Comment:
@stephenxs, can you move this to a separate PR since this change isn't related to this PR?

Reply:
Yes. It’s a dry run for me to overcome the vs test failure and check the coverage report.
Now that the vs test passed, I’m going to revert the commit

Reply:
@neethajohn 
Done. Please the rest of the PR. Thanks.

Reply:
I rebased the branch to avoid potential conflict when the fix of the vlan test issue is merged. Nothing changed in buffer stuff.